examples/js/exporters/GLTFExporter.js

```diff
@@ -1378,6 +1378,16 @@ THREE.GLTFExporter.prototype = {
 
 				if ( trackProperty === PATH_PROPERTIES.morphTargetInfluences ) {
 
+					if ( trackNode.morphTargetInfluences.length !== 1 &&
+						trackBinding.propertyIndex !== undefined ) {
+
+						console.warn( 'THREE.GLTFExporter: Skipping animation track "%s" because ' +
+							'glTF can not handle single morph animation. ' +
+							'Try to merge .morphTargetInfluences elements instead.', track.name );
+						continue;
```

Comment:
Can anyone help make this message cleaner and more straightforward?

Reply:
`glTF can not handle single morph animation` is maybe confusing? We could (technically) write `0` for the other targets, and that would be valid glTF, but I don't love that idea... maybe phrase this as `THREE.GLTFExporter: Skipping animation track "%s". Morph target keyframe tracks must target all available morph targets for the given mesh.` Perhaps this is something where the glTF spec should provide an alternative, too.